src/ui/components/Button/index.js

```diff
@@ -16,6 +27,8 @@ export default class Button extends React.Component {
     disabled: PropTypes.boolean,
     href: PropTypes.string,
     to: PropTypes.string,
+    type: PropTypes.string,
```

Comment:
I see many `type=none` already, I guess this could be a good default value in `defaultProps`, what do you think?